app/models/reservation.rb

```diff
@@ -2,15 +2,7 @@
 
 class Reservation < ApplicationRecord
   belongs_to :user
-  belongs_to :seat
+  has_many :seats
```

Comment:
This association is wrong, it should be `belongs_to :seat`

Reply:
Done